python/dgl/nn/pytorch/gt/spatial_encoder.py

```diff
@@ -138,18 +138,23 @@ def __init__(self, num_kernels, num_heads=1, max_node_type=100):
         self.num_kernels = num_kernels
         self.num_heads = num_heads
         self.max_node_type = max_node_type
-        self.gaussian_means = nn.Embedding(1, num_kernels)
-        self.gaussian_stds = nn.Embedding(1, num_kernels)
+        self.means = nn.Parameter(th.empty(num_kernels))
+        self.stds = nn.Parameter(th.empty(num_kernels))
```

Comment:
Do you need to initialize them in a particular way?

Reply:
yes, they are initialized with the uniform distribution in line 156-157